### ISSUE TITLE
estimate cost for BS sequencing of Salmon Samples

### ISSUE THREAD

**Issue by shellytrigg** [RobertsLab/resources#678]
Complete cost analysis for different methods (RRBS, WGBS, and Amplicon-based BS-seq) and include it here:

https://docs.google.com/presentation/d/1mD6ivb3xcfhvu_fF7dhYTLWytzXeYp8rheEvbTJKWnc/edit#slide=id.p

**Comment by kubu4** [comment]
@yaaminiv @sr320 I think this should be merged with this issue:

https://github.com/RobertsLab/resources/issues/543

as costs will be/are part of the decision tree.

**Comment by sr320** [comment]
Seems like there could be a universal google sheet where you input species info and everything calculates.
Maybe a Shiny app one day, but lets focus on getting some numbers first!

thanks
Steven

**Comment by kubu4** [comment]
> Seems like there could be a universal google sheet where you input species info and everything calculates.
Maybe a Shiny app one day,

Whoa! Killer idea! Hackweek 2019?

**Comment by shellytrigg** [comment]
Here's an update on where we're at with pricing:
Sea lice: using [swift (accel) library prep](https://swiftbiosci.com/accel-ngs-methyl-seq-dna-library-kit/),an estimated 240M reads would get >90% coverage of all mCpGs at 10X

**Quotes Sam got:**
Zymo: $2300/sample (i.e. $4600 for the project).
Genewiz: $2320 ($880/sample + $1440 for 1 lane of sequencing)

**Salmon:**  
[The 2017 Scientific Reports paper on temp stress methylome and transcriptome response in Atlantic Salmon](https://www.nature.com/articles/s41598-017-05222-2#additional-information) used the Zymo Methyl-miniseq protocol. I wrote to Zymo to get a quote for the following services:
Methyl-miniseq
Methyl-midiseq
Classic RRBS

**To do library prep in house:**  ~$750 not including adapter cost
Taq-alpha I = $250
MspI = $100
Zymo DNA concentrator columns (150) $150
Illumina bisulfite seq adapters
Agarose gel and gel clean up kit (qiagen QG buffer would do, $50; could alternatively do bead cleanup)
Bisulfite treatment kit (EZ DNA Methylation-Lightning Kit) $200
PCR reagents

**Comment by shellytrigg** [comment]
For all 20 salmon samlpes: **low end is ~$6000**:  RRBS MSPI/Taq-alphaI digest and clean up ($150) + NEB EM-seq kit ($895) +  1600M reads on UW's Novaseq ($4714). To do the same thing with an **in-house kit is ~$6300:** RRBS MSPI/Taq-alphaI digest and clean up ($150) + home brew kit ($1460) + 1600M reads on UW's Novaseq ($4714). **Prep with Swift Accel NGS methylseq Kit ~$6700:** RBS MSPI/Taq-alphaI digest and clean up ($150) + Swift Accel NGS methylseq Kit ($1870) + 600M reads on UW's Novaseq ($4714). **Zymo methylmini-seq service $22,100** . This is based on this info:  https://docs.google.com/spreadsheets/d/11OWceVZ9pehSSwnR_WjBQncOkRHxNLyZAR2BRNu5aVw/edit?usp=sharing , [this github issue](https://github.com/RobertsLab/resources/issues/603), [this github issue](https://github.com/RobertsLab/resources/issues/580), and https://docs.google.com/presentation/d/1mD6ivb3xcfhvu_fF7dhYTLWytzXeYp8rheEvbTJKWnc/edit#slide=id.g50f7fba360_0_22

**Comment by sr320** [comment]
@shellytrigg  - so you recommend option 2?

**Comment by shellytrigg** [comment]
@sr320 yes, if we want to follow what the Sci Reports paper did. If the gDNA yields end up being low, we can re-evaluate

**Comment by shellytrigg** [comment]
A little change up, the plan is to:

1.   Isolate DNA using the EZna extraction kit (already have in lab)

2.  RRBS MSPI/Taq-alphaI digest and clean up 

3. Library prep with Zymo Pico Methyl-seq kit (because this is the kit we are using for the sea lice samples, it will be easier to prepare everything the same way). I confirmed with Zymo that the kit should work fine with RRBS digested samples because it uses their gentlest bisulfite conversion (EZ lightening) ***as long as the UW sequencing core uses ONLY Illumina products for sequencing (no homemade reagents). I also confirmed the kit is compatible with NovaSeq flowcell technology and it's possible to multiplex all our samples (see [email about PicoMethylSeqKit_IndexAlternatives](https://github.com/RobertsLab/resources/files/3158681/PicoMethylSeqKit_IndexAlternatives.pdf)), we just need to order ~17 oligos (1 universal + 16 with unique indexes; ~10$/60bp oligo so ~$170 total)

4. Sequence on NovaSeq 6000 with SP flowcell for 1600M reads (240M for sea lice 1, 240M for sea lice 2, and 56M/salmon sample).
